core/src/main/java/org/apache/calcite/sql2rel/StandardConvertletTable.java

```diff
@@ -860,6 +861,29 @@ public RexNode convertCall(
             : operandTypeChecker.getConsistency();
     final List<RexNode> exprs = convertOperands(cx, call, consistency);
     RelDataType type = rexBuilder.deriveReturnType(op, exprs);
+
+    //generate relation for `=(Row1,Row2)`
+    if (op.kind == SqlKind.EQUALS) {
+      RexNode expr0 = RexUtil.removeCast(exprs.get(0));
+      RexNode expr1 = RexUtil.removeCast(exprs.get(1));
+      SqlKind expr0Kind = expr0.getKind();
+      SqlKind expr1Kind = expr1.getKind();
+      if (expr0Kind == SqlKind.ROW && expr1Kind == SqlKind.ROW) {
+        RexCall call0 = (RexCall) expr0;
+        RexCall call1 = (RexCall) expr1;
```

Comment:
Can you give an example for `CASE (deptno, job) WHEN (20, 'CLERK') THEN 1`, what is the `exprs.get(0)` and `exprs.get(1)`?

Is exprs.get(0) `(deptno, job` and  exprs.get(1) `(20, 'CLERK')`

Reply:
YES